### PR TITLE
Botania Support

### PR DESCRIPTION
**Pull request by Shadows-of-Fire**
Fully functional from what has been tested.  Only a single change to method parameters (Lexicon.addCraftingPage now takes a String... as recipe registry names instead of actual recipes).
Still currently prints blank lines to the crafttweaker log, would be nice to see a change to ILogger to ignore empty strings

### CODE REVIEW

**Comment by KAfable** [comment]
Hey Jared, just wondering if the pull request is good to be merged - some dev's I know are looking forward to the Botania support.

File: src/main/java/com/blamejared/compat/botania/commands/BotaniaBrewLogger.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	}
+
+	@Override
+	public void executeCommand(MinecraftServer server, ICommandSender sender, String[] arguments) {
+		for (String key : BotaniaAPI.brewMap.keySet()) {
+			CraftTweakerAPI.logCommand(key);
+		}
+
+		if (sender != null) {
+			sender.sendMessage(new TextComponentString("List generated; see crafttweaker.log in your minecraft dir"));
```
Comment (BloodWorkXGaming) [inline src/main/java/com/blamejared/compat/botania/commands/BotaniaBrewLogger.java]: I'd recommend using something like ` sender.sendMessage(SpecialMessagesChat.getLinkToCraftTweakerLog("List generated;", sender));` to get a clickable directlink to the crafttweaker.log

**Comment by Shadows-of-Fire** [comment]
There we go describes exist now

**Comment by Zer0Lyfe** [comment]
@jaredlll08 I was wondering if you knew of this pull request. Sorry for bothering you.

**Comment by belathus** [comment]
I have a request here: add support for the Orechid Ignem; the nether version of the orechid. 

https://github.com/Vazkii/Botania/blob/938aab69e5c46d782af3fdb9d647ccd754651853/src/main/java/vazkii/botania/api/BotaniaAPI.java#L628-L630

**Pull request merged**